src/proxy/momento/src/frontend.rs

```diff
@@ -86,6 +86,7 @@ pub(crate) async fn handle_resp_client(
                 let request = request.into_inner();
 
                 match request {
+                    resp::Request::BAdd(_) => todo!(),
```

Comment:
Actually, just noticed this - this introduces a panic path in a service. We should have a more defensive approach and treat it as an invalid command. I think it'd be acceptable to add a 
```_ => {
                  println!("bad request");
                  let _ = socket.write_all(b"CLIENT_ERROR\r\n").await;
                  break;
              }```

To the end of this match to catch any commands that aren't implemented.